src/plugins/vis_type_vislib/public/vislib/components/legend/_legend.scss

```diff
@@ -84,7 +84,7 @@ $visLegendLineHeight: $euiSize;
 }
 
 .visLegend__valueColorPicker {
-  width: ($euiSizeL * 8); // 8 columns
+  width: ($euiSizeL * 10); // 10 columns
```

Comment:
Is this 10 because that's the default size of oui palette?

Reply:
yep - see https://oui.opensearch.org/1.0/#/utilities/color-palettes